variables.tf

```diff
@@ -3,6 +3,11 @@ variable "naming_rules" {
   type        = string
 }
 
+variable "dnsserver" {

```

Comment:
Please rename the variable to `dns_servers`.

Reply:
done the change